web/src/App.js

```diff
@@ -49,9 +48,6 @@ function App() {
   }, []);
 
   const menuBarStyle = {
```

Comment:
Inline styling is not recommended as discussed below.

Reply:
The code was updated quickly thus I'm not sure if you had the chance to look into my comments, thus I'm writing the issue about inline styling here and resolving other issues regarding inline styling. 

I've read that, it's not a good approach to include inline styling in the jsx file. Can you move them to another CSS file ?  You can refer to [here](https://www.sitepoint.com/react-components-styling-options/) for React component styling options.